### DISCUSSION TITLE
Batch photo upscale

### DISCUSSION THREAD

**Discussion by AndreySlivka** [upscayl/upscayl#947] [Ideas]
Maybe you planned few images same time upscale ?
For example, If need more that one image upscale, 100 or more images. This time i need 100 times click "select file" and convert.

any chanse to realize interface like this ? command string or something else?

**Comment by aaronliu0130** [comment]
The "Batch Upscayl" toggle is right there.
![image](https://github.com/user-attachments/assets/797a64b6-b625-41c7-bf58-f3cd7cbdb657)

**Reply by AndreySlivka** [reply]
Thanks. I am blind  :)
